brands/amenity/bank.json

```diff
@@ -3037,6 +3037,15 @@
       "name": "First Citizens Bank"
     }
   },
+  "amenity/bank|First Citizens Bank~(USA)": {
+    "tags": {
```

Comment:
While we’re at it, we could add `1st citizens bank` as a matching name for both, similar to the `1st financial bank` in the entry below.

Reply:
Done